src/lib/exposes.ts

```diff
@@ -376,6 +376,18 @@ export class Light extends Base {
 
         return this;
     }
+
+    withColorOptions() {
```

Comment:
Can you add this as a separate expose instead of integrating it in light?

Reply:
Yes I can. By what rule do you decide where it should be placed?

Reply:
Usually wether it is part of the Home Assistant entity or not (e.g. https://www.home-assistant.io/integrations/light.mqtt/) (not always but it is a good guideline)

Reply:
0fa62e5b51e515c1fcc804e5595f15c73a8046d1